binding-service-impl/pom.xml

```diff
@@ -29,5 +29,11 @@
 			<artifactId>esignet-core</artifactId>
 			<version>${esignet.core.version}</version>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>io.mosip.esignet</groupId>
+            <artifactId>oidc-service-impl</artifactId>
+            <version>1.5.1-SNAPSHOT</version>
```

Comment:
please use a property here, Rather than hardcoding the version.

Reply:
Added ${project.version} instead of hardcoded value.